osf/models/metaschema.py

```diff
@@ -40,7 +43,7 @@ def get_latest_version(self, name, only_active=True):
         if sorted_schemas:
             return sorted_schemas.last()
         else:
-            return None
+            return self.none()
```

Comment:
What is this?

Reply:
It returns an empty queryset, it's necessary because we concatenate querysets on [this line](https://github.com/CenterForOpenScience/osf.io/pull/9282/files#diff-7ff9bf7f628fdb6021ea91220da7518dR72).